### ISSUE TITLE
Docs: "common" js files are copied to every docs version.

### ISSUE THREAD

**Issue by jansiegel** [handsontable/handsontable#9528]
### Description
I'm not sure if it's a bug or was it designed to work like that, but some things regarding the handling of "common" `js` files in the documentation seem inconsistent to me.

1. There are some javascript files in `.vuepress/public/scripts` directory - in this case I'm talking mostly about:
    - `fixer.js`
    - `helpers.js`
2. The `scripts` directory, being a part of `public` is being copied to every versioned documentation directory, so there's a `scripts` folder inside every:
    - `docs/`
    - `docs/[version]`
    directory.
3. The `fixer.js` and `helper.js` files are being treated as "common" for all versions of the documentation and they're always being fetched from the _latest version_ of documentation for _every_ versioned docs.
The code responsible for that logic is being stored in [`getCommonScript` in `.vuepress/handsontable-manager/dependencies.js`](https://github.com/handsontable/handsontable/blob/12.0.1/docs/.vuepress/handsontable-manager/dependencies.js#L22-L29)
Basically, every version of docs pulls those files from `/docs/scripts/`.
4. That makes the `scripts` copies in `docs/[version]` redundant if I understand this logic correctly and am not missing anything.

IMO it's either wrong that the `scripts` directory is being cloned to every `docs/[version]` or it's wrong that those helper files are not versioned at all - I'm leaning towards the latter, as now changing anything in `fixer.js` modifies the way _every_ version of the documentation works.

@warpech, I need your opinion on this.

PS. This gets even more convoluted after #9441, as every _version_ and _framework_ "flavor" of the documentation will have to pull those files from, most probably, `docs/javascript-data-grid/scripts`?

**Comment by warpech** [comment]
We discussed that on Slack: https://handsoncode.slack.com/archives/C034WPQUG9W/p1653550919412959

**Comment by jansiegel** [comment]
As the documentation structure changed, the description of this issue is no longer accurate.
